### ISSUE TITLE
Bridge should have a health check endpoint

### ISSUE THREAD

**Issue by phutchins** [Storj/bridge#329]
### Package Versions

Storj Bridge Version
```
storj-bridge@4.2.2 /opt/storj/bridge
└── (empty)
```

NodeJS Version

```
v6.9.1
```

### Expected Behavior

An endpoint should exist which returns response codes that indicate the instances health and ability to respond to requests. This would be used in a loadbalancer to determine if the instance is ready to receive traffic.

For instance, if the instance had no DB connection, a reasonable response could look something like the following.

```
$ curl -i https://api.storj.io/health
HTTP/1.1 503 Service Unavailable
```

### Actual Behavior

No health check exists.

**Comment by dylanlott** [comment]
Addressed by #411

**Comment by braydonf** [comment]
Let's close when merged.
